curriculum/challenges/english/25-front-end-development/workshop-reusable-mega-navbar/6740495ba48aa94e5667b436.md

```diff
@@ -34,10 +34,10 @@ You should use the `export` keyword to export the `Navbar` component.
 assert.match(code, /export\s+(const|function)\s+Navbar\s*(=\s*)?\(\)\s*(=>\s*)?\{\s*/);
 ```
 
-You should return an empty pair of round parentheses inside the `Navbar` function.
+You should return nothing inside the `Navbar` function.
```

Comment:
we shouldn't mention return at all because it is not being used. This step is to just create and export the functional component. 

```suggestion
You should export the `Navbar` component.
```